Plots/Contours/NCL_conwomap_3.py

```diff
@@ -0,0 +1,92 @@
+"""
+NCL_conwomap_3.py
+=================
+Concepts illustrated:
+  - Drawing a simple contour plot
+  - Generating dummy data using "random_normal"
+  - Masking mirrored contour data
+  - Drawing a perimeter around areas on a contour plot with missing data
+  - Turning off the bottom and right borders of a contour plot
+  - Using "getvalues" to retrieve resource values
+  - Changing the labels and tickmarks on a contour plot
+  - Adding a complex Greek character to a contour plot
+  - Moving the contour informational label into the plot
+  - Forcing tickmarks and labels to be drawn on the top X axis in a contour plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/conwomap_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/conwomap_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Generate random data:
+
+xlist = np.linspace(0, 30.0, 30)
+ylist = np.linspace(0, 30.0, 30)
+xdata, ydata = np.meshgrid(xlist, ylist)
+
+zdata = np.random.normal(0, 2, size=(30, 30))
```

Comment:
Ditto, the second parameter (variance?)  for random.normal is 2, in the NCL version it is 3.0. They should be consistent. If there is a reason that they need to be different it should be documented.